pkg/test/echo/cmd/client/main.go

```diff
@@ -118,6 +119,8 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&caFile, "ca", "/cert.crt", "CA root cert file")
 	rootCmd.PersistentFlags().StringVar(&msg, "msg", "HelloWorld",
 		"message to send (for websockets)")
+	rootCmd.PersistentFlags().BoolVar(&http2, "http2", false,
```

Comment:
Currently we have no way for the client to send h2, so this enables that.

Reply:
TIL there is explicit standard for "prior knowledge"